tools/jenkins/run_full_performance.sh

```diff
@@ -52,4 +52,12 @@ tools/run_tests/run_performance_tests.py \
     --remote_worker_host grpc-performance-server-32core grpc-performance-client-32core grpc-performance-client2-32core \
     || EXIT_CODE=1
 
+# selected scenarios on Windows
```

Comment:
I take it this is more experimental right now? Should these maybe run on a separate job? Though either is ok with me